README.md

```diff
@@ -185,7 +192,6 @@ jaffle_shop:
 
 and store this file in project root `./profiles.yml`.
 
-
 ### Create new workflow
 
 - inside your Github repository create folder `.github/workflows` (unless it already exists).
```

Comment:
@BAntonellini this is showing it configured to run only on pushes to main, can you see how we have it in Balboa?  maybe we change this to pr.yml and have it run on pr creation on any branch.